### PR TITLE
bugfix: remove replaces for chunk_html to show newlines properly

### PR DESCRIPTION
**Pull request by aaryanpunia**

### CODE REVIEW

**Review by skeptrunedev — CHANGES_REQUESTED** [review]
Need to add repro steps to the PR so this can be tested efficiently during review

**Comment by aaryanpunia** [comment]
```
fetch("http://localhost:8090/api/chunk", {
  "headers": {
    "accept": "*/*",
    "accept-language": "en-US,en;q=0.9",
    "content-type": "application/json",
    "sec-ch-ua": "\"Not/A)Brand\";v=\"8\", \"Chromium\";v=\"126\", \"Google Chrome\";v=\"126\"",
    "sec-ch-ua-mobile": "?0",
    "sec-ch-ua-platform": "\"macOS\"",
    "sec-fetch-dest": "empty",
    "sec-fetch-mode": "cors",
    "sec-fetch-site": "same-site",
    "tr-dataset": "4947b8ce-87e1-4876-a943-283e9c6d46cb",
    "x-api-version": "2.0"
  },
  "referrer": "http://localhost:5174/",
  "referrerPolicy": "strict-origin-when-cross-origin",
  "body": "{\"chunk_html\":\"newline one\\nnewlinetwo\\n\\n\"}",
  "method": "POST",
  "mode": "cors",
  "credentials": "include"
});
```

File: frontends/search/src/components/ChunkMetadataDisplay.tsx [hunk 1]
```diff
@@ -314,10 +314,6 @@ const ChunkMetadataDisplay = (props: ChunkMetadataDisplayProps) => {
             innerHTML={sanitizeHtml(
               props.chunk.chunk_html !== undefined
                 ? props.chunk.chunk_html
```
Comment (skeptrunedev) [inline frontends/search/src/components/ChunkMetadataDisplay.tsx:316]: I believe this should get the same replace as in the `ScoreChunk` file. Will ship it real quick.

**Review by skeptrunedev — CHANGES_REQUESTED** [review]

**Comment by skeptrunedev** [comment]
Added `replaceAll("\n", "<br>")` to `ChunkMetadata` and `ScoreChunk` for the chat playground. Also squashed this down to one commit with a better message. Ready to merge!

**Review by skeptrunedev — APPROVED** [review]

**Pull request merged**